update-timescaledb.md

```diff
@@ -0,0 +1,35 @@
+# Updating TimescaleDB versions [](update)
+
+This section describes how to upgrade between different versions of
+TimescaleDB. TimescaleDB supports **in-place updates**:
+you don't need to dump and restore your data, and versions are published with
+automated migration scripts that convert any internal state if necessary.
```

Comment:
Do we want to explicitly say we currently only support forward updates / upgrades, not backwards / downgrades?